README.md

```diff
@@ -1,18 +1,18 @@
 docker/migrator
 =================
 
-Tool to migrate Docker images from Docker Hub or v1 registry to a v2 registry
+Tool to migrate Docker images from Docker Hub or v1 registry to a v2 registry including AWS ECS Repositories
 
-https://hub.docker.com/r/docker/migrator/
+https://hub.docker.com/r/mattnowina/migrator/
 
 ## Usage
 
 ```
-docker run -it \
-  -v /var/run/docker.sock:/var/run/docker.sock \
-  -e V1_REGISTRY=v1.registry.fqdn \
-  -e V2_REGISTRY=v2.registry.fqdn \
-  docker/migrator
+docker run --rm -it -v ~/.aws:/root/.aws:ro -v /var/run/docker.sock:/var/run/docker.sock -e V1_REGISTRY=v1.registry.fqdn -e V2_REGISTRY=v2.registry.fqdn mattnowina/migrator
```

Comment:
This should be switched back so it is easy to read due to line going off the screen.  If this basic example could be left intact, that would be preferred so the basic usage is available but would be great to have an examples section below the `Prerequisites` section that show the specific usage for specific situations such as with ECR.